tools/pipeline_perf_test/test_suites/integration/configs/integration_report_logs.yaml

```
@@ -0,0 +1,323 @@
+queries:
+  - name: Filter superflous metadata
+    sql: |
+      CREATE VIEW metadata_filtered AS
+      SELECT *
+      FROM metadata
+      WHERE Attribute NOT LIKE 'test.ctx.%';
+
+  - name: Calculate observation window for this test
+    sql: |
+      CREATE TABLE observation_window AS WITH event_times AS (
+        SELECT
+          "attributes.test.name" AS test_name,
+          name AS event_name,
+          TO_TIMESTAMP(timestamp / 1000000000.0) AS timestamp,
```

I think we need to remove the trailing comma.

```suggestion
          TO_TIMESTAMP(timestamp / 1000000000.0) AS timestamp
```